### PR TITLE
Adding Restlet Framework to the REST frameworks

### PR DESCRIPTION
**Pull request by glaforge**

### CODE REVIEW

**Comment by geoand** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -462,6 +462,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Jersey](https://jersey.java.net/) - JAX-RS reference implementation.
 * [RESTEasy](http://resteasy.jboss.org/) - Fully certified and portable implementation of the JAX-RS specification.
 * [RestExpress](https://github.com/RestExpress/RestExpress) - Thin wrapper on the JBOSS Netty HTTP stack to provide scaling and performance.
+* [Restlet Framework](http://restlet.com/products/restlet-framework/) - The flexible and powerful pioneer REST Web API framework for Java.
```
Comment (akullpp) [inline README.md]: Please remove the article from the beginning for consistency with other entries. Also remove the "for Java" it is implied.

What does "pioneer" mean?

Why is "Web" capitalized?

Comment (glaforge) [inline README.md]: Pioneer because that was the first REST frameworks available 10 years or so ago.
I often use Web capitalized, but I can change this.

Comment (akullpp) [inline README.md]: Please do this, remove [for Java] and lowercase web.

**Comment by akullpp** [comment]
Added in c7e2aa7

**Pull request closed without merging**